site/content/community/get-involved.md

```diff
@@ -1,7 +1,9 @@
 ---
-title: Community
-type: docs
-menu: docs
+title: Get Involved !
+weight: 1
+type: community
+menu: "community"
+aliases: ["/community/"]
 ---
 
 # Community
```

Comment:
Remove since the page title already gets rendered as an `h1`

Reply:
the weighting and aliases ensure that the get involved page is rendered at the top level from the nav link

Reply:
sorry, I meant something different. The `# Community` in markdown gets rendered as an `h1` HTML element, but Hugo inserts another `h1` based on the `title: Get Involved !`. So the page ends up with two headers which looks odd. On other pages we don't include a single pound header.

Reply:
Ah yes, I already removed the # Community